makejdk-any-platform.1

```diff
@@ -53,3 +53,9 @@ use ssh when cloning git
 .BR \-dsgc ", " \-\-disable-shallow-git-clone
 disable shallow cloning of git repo(s) using the --depth=1 CLI option
 .TP
+.BR \-fm ", " \-\-freemarker
+fetch freemarker jar file and add --with-freemarker-jar option to configure command
+.TP
+.BR \-gs ", " \-\-get-source
```

Comment:
Perhaps add a comment that this will get removed later and maybe raise a GitHub issue?

Reply:
i can do that, sure.